os_migrate/roles/export_projects/README.md

```diff
@@ -0,0 +1,2 @@
+Please refer to
```

Comment:
This should point similar to: https://github.com/os-migrate/os-migrate/pull/270/files#diff-c3d7c71d6d79498900947637e86723ddL2